### PR TITLE
enable telco bastion from step registry

### PR DESCRIPTION
**Pull request by yuvalk**

### CODE REVIEW

File: ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh [hunk 1]
```diff
@@ -30,10 +32,90 @@ cat << EOF > ~/ocp-install.yml
   - name: Clone repo
     git:
       repo: https://github.com/karmab/kcli-openshift4-baremetal.git
-      dest: "~/kcli-openshift4-baremetal"
+      dest: ~/kcli-openshift4-baremetal
       version: master
       force: yes
     retries: 5
+  - name: Add master workaround manifest
```
Comment (yuvalk) [inline ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh:39]: @johnbieren 
I think we need to discuss whether we want to split the setup step into multiple steps that make sense

Comment (johnbieren) [reply ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh:40]: Sure, whatever you think is best, you know a lot more about the tests than I do. We will have to make sure we split it in a way that doesn't make us have to reprovision the cluster every time, though (unless you want to reprovision each time)

**Pull request merged**